services/static-webserver/client/source/class/osparc/study/PricingUnitLicense.js

```diff
@@ -29,6 +29,13 @@ qx.Class.define("osparc.study.PricingUnitLicense", {
       nullable: true,
       event: "changeShowRentButton"
     },
+
+    licenseUrl: {
+      check: "String",
+      init: false,
+      nullable: true,
+      event: "changeLicenseUrl"
```

Comment:
This will not be one but many. Every model will have a different license.  
Check https://github.com/ITISFoundation/licenses/tree/main/models

The way i understand it is that for every licensed_item you will get that link using 
![image](https://github.com/user-attachments/assets/4f8be5ff-6286-42d1-8307-0bcc8d3ebcbf)

This will only be available once the VIP API exposes that information and the backend syncer catches up. In the meantime, it could be added manually in the `licensed_resource_data` column.

Reply:
I agree that this should go at model level, but right now we have it on the licesedItem level. Once the backend makes the change I'll update the frontend. In the meantime, let's move forward to at least have all the elements in place.